packages/@o3r/telemetry/package.json

```diff
@@ -22,11 +22,18 @@
     "@angular-devkit/architect": "~0.1802.0",
     "@angular-devkit/core": "~18.2.0",
     "@angular-devkit/schematics": "~18.2.0",
+    "minimist": "^1.2.6",
```

Comment:
should this be installed with the `ng-add` of `@o3r/telemetry`?

Reply:
as discussed moved as dep